internal/wasm/jit/jit_amd64.go

```diff
@@ -90,12 +90,12 @@ func newArchContext() (ret archContext) { return }
 
 // newCompiler returns a new compiler interface which can be used to compile the given function instance.
 // Note: ir param can be nil for host functions.
```

Comment:
I'd appreciate it if the doc explains what the caller must do against the returned func value